<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>330</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>330</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>330</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Servicio ¿Cuando llega? del ETR</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/res/img/icon.png</normaloff>:/res/img/icon.png</iconset>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_consulta">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>371</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Consultar horario</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>51</y>
       <width>321</width>
       <height>160</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,3">
        <item>
         <widget class="QLabel" name="label_lineas">
          <property name="text">
           <string>Línea:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_linea">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
          <property name="currentIndex">
           <number>-1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
        <item>
         <widget class="QLabel" name="label_calle">
          <property name="text">
           <string>Calle:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_calle">
          <property name="editable">
           <bool>false</bool>
          </property>
          <property name="currentIndex">
           <number>-1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,3">
        <item>
         <widget class="QLabel" name="label_interseccion">
          <property name="text">
           <string>Intersección:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_interseccion">
          <property name="editable">
           <bool>false</bool>
          </property>
          <property name="currentIndex">
           <number>-1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,3">
        <item>
         <widget class="QLabel" name="label_parada">
          <property name="text">
           <string>Parada:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_parada">
          <property name="editable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QCheckBox" name="checkBox_consultaBatch">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>222</y>
       <width>111</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Consulta batch</string>
     </property>
     <property name="shortcut">
      <string>Ctrl+B</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_consultar">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>222</y>
       <width>91</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Consultar</string>
     </property>
     <property name="shortcut">
      <string>Ctrl+C</string>
     </property>
    </widget>
    <zorder>layoutWidget</zorder>
    <zorder>checkBox_consultaBatch</zorder>
    <zorder>pushButton_consultar</zorder>
    <zorder>pushButton_consultar</zorder>
   </widget>
   <widget class="QTabWidget" name="tabWidget_resultados">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>341</width>
      <height>261</height>
     </rect>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_resultados">
     <property name="accessibleName">
      <string/>
     </property>
     <attribute name="title">
      <string>Resultado</string>
     </attribute>
     <widget class="QPlainTextEdit" name="plainTextEdit_resultados">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>311</width>
        <height>211</height>
       </rect>
      </property>
      <property name="undoRedoEnabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_historico">
     <attribute name="title">
      <string>Histórico</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_resultados">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>311</width>
        <height>211</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectItems</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="columnCount">
       <number>2</number>
      </property>
      <column>
       <property name="text">
        <string>Fecha / hora</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Resultado</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionAcercaDe"/>
   </widget>
   <widget class="QMenu" name="menuInicio">
    <property name="title">
     <string>Inicio</string>
    </property>
    <widget class="QMenu" name="menu_ultimas_consultas">
     <property name="title">
      <string>Últimas consultas</string>
     </property>
     <addaction name="actionConsulta_1"/>
     <addaction name="actionConsulta_2"/>
     <addaction name="actionConsulta_3"/>
    </widget>
    <addaction name="menu_ultimas_consultas"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuRegistro">
    <property name="title">
     <string>Registro</string>
    </property>
    <addaction name="actionAbrir_archivo_de_registro"/>
    <addaction name="actionAnalizar_archivo_de_registro"/>
   </widget>
   <addaction name="menuInicio"/>
   <addaction name="menuRegistro"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
  </widget>
  <action name="actionAcercaDe">
   <property name="text">
    <string>Acerca de...</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="actionAbrir_archivo_de_registro">
   <property name="text">
    <string>Ver  archivo de registro...</string>
   </property>
  </action>
  <action name="actionAnalizar_archivo_de_registro">
   <property name="text">
    <string>Analizar archivo de registro...</string>
   </property>
  </action>
  <action name="actionConsulta_1">
   <property name="text">
    <string>Consulta 1</string>
   </property>
  </action>
  <action name="actionConsulta_2">
   <property name="text">
    <string>Consulta 2</string>
   </property>
  </action>
  <action name="actionConsulta_3">
   <property name="text">
    <string>Consulta 3</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
